README.md

```diff
@@ -34,39 +34,15 @@
 </details>
 
 ### Key Features <a name="key-features"></a>
-
-- OOP school library: decorate a class
 - OOP school library: set up associations
 - OOP school library: add basic UI
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
-
-## 🚀 You can check the Live Demo here<a name="live-demo"></a>
-
-The live demo will be availabe soon
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
 <!-- GETTING STARTED -->
 
 ## 💻 Getting Started <a name="getting-started"></a>
 
-- Step 1: Clone this project using this git comand "git clone", (if you don't have git installed please visit
-  https://git-scm.com/downloads and install it).
-- Step 2: Once you have installed git you can clone this project using the command below:
-  ```
-  git clone https://github.com/Gardimy/OOP-school-library
-  ```
-- Step 3: Once you have cloned this repo navigate to the folder that contains
-  the repository.
-- Step 4: If you want to watch the html file open it with your favorite web browser(This project MUST be opened from VS Code for full functionality).
-- Step 5: If you want to edit this project open it with your favorite text editor.
-
-<p align="right">(<a href="#readme-top">back to top</a>)</p>
-
-## 💻 Getting Started <a name="getting-started"></a>
-
 To get a local copy up and running cloning this repo `https://github.com/Gardimy/OOP-school-library`.
 
 ### Prerequisites
```

Comment:
- [ ] Good job trying to implement some of changes I mentioned in the previous review. However as I pointed, kindly ensure that you update the current some `sub-sections` of the main `Getting Started` section with correct and appropriate content information such as how to `access` the cloned directory, `install` bundles(if necessary), `run/use` your app, `run tests`,... To help you, I suggest you refer **[this gist_file](https://gist.github.com/tresorsawasawa/0eba0570695aef068a7837c349f94737)**, as it contains all information you need. By correctly complete this section, it will ensure that, even non technical users can run easily run your project.